### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by ski232323**
# new changes on 28/11
- new exercices

### CODE REVIEW

File: dictionnaire.py [hunk 1]
```diff
@@ -1,9 +1,16 @@
-# Debut 
-voitures = {
```
Comment (ski232323) [inline dictionnaire.py:2]: # old

**Review by ski232323 — COMMENTED** [review]
# everything is ok

File: dictionnaire.py [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+employees={
```
Comment (ski235) [inline dictionnaire.py:1]: ok

**Review by ski235 — APPROVED** [review]
ok

**Pull request merged**